### PR TITLE
New package: KsplDev.Net8TestApp version 1.0.0.0

### PR DESCRIPTION
**Pull request by KsplDev**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/185337)

### CODE REVIEW

**Comment by KsplDev** [comment]
@microsoft-github-policy-service agree company="KsplDev"

**Comment by KsplDev** [comment]
@microsoft-github-policy-service agree

@microsoft-github-policy-service agree company="KsplDev"

File: manifests/k/KsplDev/Net8TestApp/1.0.0.0/KsplDev.Net8TestApp.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: KsplDev.Net8TestApp
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: KsplDev
+PackageName: Net8TestApp
+License: OPENS
+ShortDescription: Test App Net 8
```
Comment (Trenly) [inline manifests/k/KsplDev/Net8TestApp/1.0.0.0/KsplDev.Net8TestApp.locale.en-US.yaml:10]: Please provide a better description of what this app is, what it does, or what it is used for

File: manifests/k/KsplDev/Net8TestApp/1.0.0.0/KsplDev.Net8TestApp.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KsplDev.Net8TestApp
+PackageVersion: 1.0.0.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Net8TestApp.exe
+Installers:
+- InstallerUrl: https://github.com/KsplDev/Net8TestApp/releases/download/TestRelease/Net8TestApp.zip
```
Comment (Trenly) [inline manifests/k/KsplDev/Net8TestApp/1.0.0.0/KsplDev.Net8TestApp.installer.yaml:11]: I don't think a `TestRelease` makes sense to put into WinGet. Is there a stable release that isn't a test release?

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**